docs/user-guide/mvd-configuration.md

```diff
@@ -43,7 +42,7 @@ Follow these optional steps to configure the default connection to open for the
     type: <"telnet" or "tls">
   }
 ```
-### Setting up the VT Terminal application plug-in
+### Setting up the VT Terminal application plugin
```

Comment:
It seems like some parts of the doc refer to "plug-ins" and some "plugins" I am changing everything to plugins to match what normally people use and what I see common in the community. Lmk if this is bad